tests/unit/libbpf_test.cpp

```diff
@@ -166,6 +166,8 @@ TEST_CASE("libbpf map", "[libbpf]")
     REQUIRE(bpf_map__prev(map, object) == nullptr);
     REQUIRE(bpf_map__prev(nullptr, object) == map);
 
+    const char* name = bpf_map__name(map);
+    REQUIRE(name == nullptr); // droppacket.o has no map name.
```

Comment:
Just mentioning: PR for issue #81 will populate map names also from ELF file.

Reply:
that will be great.  That PR will need to update this line since the test will fail until this line is updated to match.